### ISSUE TITLE
Switch between WebIDs fails

### ISSUE THREAD

**Issue by EnnoMeijers** [netwerk-digitaal-erfgoed/solid-crs#491]
Starting with a normal login session with my (Inrupt) WebID and authorizing the app to approach my pod through the IdP. If I logout from the app and login with an other (Inrupt) WebID the data shown is still coming from the pod of the first WebID. Only after removing the cookies from inrupt.com I am able to switch between pods.

**Comment by ddeboer** [comment]
Duplicate of #223?
